contrib/mungegithub/blunderbuss.yml

```diff
@@ -0,0 +1,60 @@
+prefixMap:
+  test:
+    - ixdy
+
+  pkg/admission:
+    - lavalamp
+    - smarterclayton
+
+  pkg/api:
+    - bgrant0607
+
+  pkg/apiserver:
+    - lavalamp
+    - smarterclayton
+
+  pkg/authentication:
+    - liggitt
+    - erictune
+
+  pkg/client:
+    - lavalamp
```

Comment:
Oh, sorry.  I meant, deads2k would be appropriate for this package because
we've been in it a lot and have strong opinions in how to keep it clean.

On Mon, Aug 10, 2015 at 4:06 AM, David Oppenheimer <notifications@github.com
> wrote:

> In contrib/mungegithub/blunderbuss.yml
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/12365#discussion_r36609256>
> :
>
> > +    - lavalamp
> > +    - smarterclayton
> > +
> > +  pkg/api:
> > +    - bgrant0607
> > +
> > +  pkg/apiserver:
> > +    - lavalamp
> > +    - smarterclayton
> > +
> > +  pkg/authentication:
> > +    - liggitt
> > +    - erictune
> > +
> > +  pkg/client:
> > +    - lavalamp
>
> I was just responding to your use of "Cc" in "Cc me and Deads2k" so you
> were clear that this isn't actually a Cc. I wasn't suggesting to not add
> the people you suggested. Sorry for being unclear.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/12365/files#r36609256>
> .
>



-- 
Clayton Coleman | Lead Engineer, OpenShift